engine/src/main/java/org/destinationsol/game/ship/FarShip.java

```diff
@@ -189,4 +190,24 @@ public float getMoney() {
     public ItemContainer getIc() {
         return container;
     }
+
+    /**
+     * Each FarShip could be a mercenary. Each mercenary is definitely a FarShip .
```

Comment:
This really should not be in the method javadoc, but rather class javadoc. Also, javadoc should start with one sentence summing the rest of the javadoc, ended by dot and a newline